bazel_tools/scala.bzl

```diff
@@ -135,7 +133,6 @@ plugin_scalacopts = [
 # []
 # ^ means nothing to remove
 lf_scalacopts = [
-    "-Ywarn-unused",
 ]
```

Comment:
Are we keeping `lf_scalacopts` around for a reason?

Reply:
It turned out to be very useful, since it made it possible to fully migrate part of the codebase to the flag (in fact, I wish there were more partitions of the codebase with cross-project scalacopts sets like this one), it would be a hassle to remove, and it would be a hassle to restore.

Removing it would be like saying about a Scala build that supported cross-version builds "well, we support the latest Scala version, and it's not like they're ever going to release a new Scala version, right?"